client/src/featureform/register.py

```diff
@@ -20,14 +20,18 @@
 from .tls import insecure_channel, secure_channel
 from .resources import ResourceState, Provider, RedisConfig, FirestoreConfig, CassandraConfig, DynamodbConfig, \
     MongoDBConfig, PostgresConfig, SnowflakeConfig, LocalConfig, RedshiftConfig, BigQueryConfig, SparkConfig, \
-    AzureFileStoreConfig, OnlineBlobConfig, K8sConfig, User, Location, Source, PrimaryData, SQLTable, \
+    AzureFileStoreConfig, OnlineBlobConfig, K8sConfig, S3StoreConfig, User, Location, Source, PrimaryData, SQLTable, \
     SQLTransformation, DFTransformation, Entity, Feature, Label, ResourceColumnMapping, TrainingSet, ProviderReference, \
-    EntityReference, SourceReference, ExecutorCredentials, ResourceRedefinedError, ResourceStatus, Transformation
+    EntityReference, SourceReference, ExecutorCredentials, ResourceRedefinedError, ResourceStatus, Transformation, AWSCredentials
 
 from .proto import metadata_pb2_grpc as ff_grpc
 
+
 NameVariant = Tuple[str, str]
 
+s3_config = S3StoreConfig("", "", AWSCredentials("id", "secret"))
```

Comment:
Is there a way to do this without having to create the actual object?

Reply:
We can hardcode the types but we will have to modify it if we modify the type.

Reply:
Does it make sense to make all the types an enum?